pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -1,3 +1,4 @@
+"CheckAttach"
```

Comment:
Well, I must admit that I was a little confused by the instructions of the [wiki](https://nixos.org/wiki/Vim_plugins) and not really sure about what I was doing during the whole process.

The plugin names were taken via the `:AddonInfo` command in vim (`:AddonsInfo CheckAttach` and `:AddonsInfo vimwiki`) and the derivations were generated by following the instructions in [vim-utils.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/vim-plugins/vim-utils.nix).

Sorry if the pull-request doesn't make any sense. :)